_artifacts/data-prepper/data-prepper-1.3.0-linux-x64.markdown

```diff
@@ -0,0 +1,12 @@
+---
+role: ingest
+artifact_id: data-prepper
+version: data-prepper-1.3.0
+platform: linux
+architecture: x64
+type: targz
+artifact_url: https://artifacts.opensearch.org/data-prepper/1.2.1/opensearch-data-prepper-jdk-1.3.0-linux-x64.tar.gz
```

Comment:
The filename is correct using `1.3.0`, but you still have `1.2.1` in the path leading up to it. The directory structure does have the version in the name and we need to update accordingly.

Reply:
I'll update it. Thanks!